File: Zad3/app/src/main/java/com/example/bmizad2/ui/quiz/QuizViewModel.java
package com.example.bmizad2.ui.quiz;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class QuizViewModel extends ViewModel {
    private final MutableLiveData<String> mText;

    public QuizViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Challenge yourself");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: Zad4/app/src/main/java/com/example/bmizad2/ui/quiz/QuizViewModel.kt
package com.example.bmizad2.ui.quiz

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class QuizViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()

    init {
        mText.value = "Challenge yourself"
    }

    val text: LiveData<String>
        get() = mText
}